### ISSUE TITLE
In Windows High Contrast, DropDown's Options have no focus outline and the chevron is difficult to see 

### ISSUE THREAD

**Issue by Paul-Grogan** [carbon-design-system/carbon#6921]
## Environment

> Operating system

Windows 10

> Browser

Edge - Version 85.0.564.63 (Official build) (64-bit)

> Assistive technology used to verify

Windows High Contrast Mode enabled

## Detailed description

> What version of the Carbon Design System are you using?

Unknown, assuming the latest because this was observed on the Sample pages

> What did you expect to happen?

A focus indicator to appear around the currently focused Option

> What happened instead?

There is no visible indicator to let the user know what option they are currently focused on.

> What WCAG 2.1 checkpoint does the issue violate?

Success Criterion 2.4.7: Focus Visible - https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

## Steps to reproduce the issue

1. Step one
Enable Windows 10's High Contrast mode

2. Step two
Launch (Edge Chromium) and navigate to the Live Demo section of the Dropdown sample
https://www.carbondesignsystem.com/components/dropdown/usage/#live-demo

3. Step three
Examine the Chevron, it will be there, but very difficult to see.

4. Step four
Using the Keyboard, place focus onto the Dropdown, press the down arrow to open the options list. 
Continue to press Down to move through the list, no focus indicator will be presented to show what option is currently selected.

## Additional information

- Screenshots or code
![000737](https://user-images.githubusercontent.com/70535014/94424408-15ea8780-0182-11eb-8574-876dc2143b51.png)

**Comment by tw15egan** [comment]
Verified that this is working now with the latest updates
